libraries/chain/contracts/chain_initializer.cpp

```diff
@@ -150,19 +150,19 @@ abi_def chain_initializer::eos_contract_abi(const abi_def& eosio_system_abi)
 
    eos_abi.structs.emplace_back( struct_def {
       "passrecovery", "", {
-         {"account", "account_name"},
```

Comment:
Why are we changing the type for account to be "name"?  I know that technically it is but I think this definition should match the definition provided in types.hpp, to make it easier to manually maintain these definitions that need to match.

Reply:
That was an oversight, I had changed it's field before and changed it back.  Will change that back to just name.